.circleci/config.yml

```diff
@@ -1,18 +1,18 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@4.8.1
+  architect: giantswarm/architect@4.21.0
 
 workflows:
   package-and-push-chart-on-tag:
     jobs:
       - architect/push-to-app-catalog:
           context: "architect"
-          # executor: "app-build-suite" # uncomment this if you want automatic metadata generation and helm chart linting
-          name: "package and push {APP-NAME} chart"
-          app_catalog: "giantswarm-playground-catalog"
-          app_catalog_test: "giantswarm-playground-test-catalog"
-          chart: "{APP-NAME}"
+          executor: "app-build-suite"
+          name: "push-to-default-app-catalog"
+          app_catalog: "default-catalog"
+          app_catalog_test: "default-test-catalog"
+          chart: "cluster-api-provider-cloud-director-app"
           # Trigger job on git tag.
           filters:
             tags:
-              only: /^v.*/
+              only: /^v.*/
```

Comment:
we should keep in mind until we don't use the `architect/push-to-app-collection` job, we have to do the updates to new app versions by our own. I think it's ok for now but we have to discuss this behavior anyhow (because CAPO app is currently configured to do automated app updates after a release got created but CAPI app is configured to do it not that way)

```yaml
      - architect/push-to-app-collection:
          name: openstack-app-collection
          context: "architect"
          app_name: "cluster-api-provider-openstack"
          app_collection_repo: "openstack-app-collection"
          requires:
            - push-to-control-plane-app-catalog
          # Trigger job on git tag.
          filters:
            branches:
              ignore: /.*/
            tags:
              only: /^v.*/
```